e2e/_suites/ingest-manager/features/agent_endpoint_integration.feature

```diff
@@ -0,0 +1,38 @@
+@agent_endpoint_integration
+Feature: Agent Endpoint Integration
+  Scenarios for Agent to deploy Endpoint and sending data to Ingest Manager and Elasticsearch.
+
+@deploy-endpoint-with-agent
+Scenario: Adding the Endpoint Integration to an Agent makes the host to show in Security App
+  Given a "centos" agent is deployed to Fleet
+    And the agent is listed in Fleet as "online"
+  When the "Elastic Endpoint" integration is "added" in the "default" policy
+  Then the "Elastic Endpoint" datasource is shown in the "default" policy as added
+    And the host name is shown in the Administration view in the Security App as "online"
+    
+@endpoint-policy-check
+Scenario: Deploying an Endpoint makes policies to appear in the Security App
+  When an Endpoint is successfully deployed with a "centos" Agent
+  Then the policy response will be shown in the Security App
+
+@set-policy-and-check-changes
+Scenario: Changing an Agent policy is reflected in the Security App
+  Given an Endpoint is successfully deployed with a "centos" Agent
```

Comment:
This step executes there three tasks in a row:
1. deploy a container from the "image", installing the proper agent in there
2. create an enrollment token and enrol the agent in Kibana
3. add the Endpoint integration to the default policy (old configuration)